DOCS-9196 PR to move docs from working branch to prod.

sleslie23 opened the pull request.

Commit:
DOCS-9309 adding structure for rtf 1.8

Commit:
Merge pull request #304 from mulesoft/DOCS-9309-create-1.8-organization

DOCS-9309 adding structure for rtf 1.8

Commit:
DOCS-9309 adding files from beta branch.

Commit:
Merge pull request #306 from mulesoft/DOCS-9309-create-1.8-organization

DOCS-9309 adding files from beta branch.

Commit:
Adding top-level nodes for appliance and selfmanaged.

Commit:
Updating nav.adoc.

Commit:
DOCS-9203 adding support matrix. Moving content to appliance-specific topic.

Commit:
DOCS-9203 minor edits and clarification.

Commit:
Updating release number.

Commit:
Merge pull request #308 from mulesoft/DOCS-9203-add-feature-matrix

DOCS-9203 Revising overview and adding feature support matrix

Commit:
DOCS-9203 incorporating comments from Sundip.

Commit:
Merge pull request #309 from mulesoft/DOCS-9203-add-feature-matrix

DOCS-9203 incorporating comments from Sundip.

Commit:
Updating name for RTF appliance.

Commit:
Removing old beta files.

Commit:
Removing old beta links.

Commit:
Update nav.adoc.

Commit:
DOCS-9207 adding hosts, ports, and certs info.

Commit:
Minor edit.

Commit:
Renaming beta file.

Commit:
DOCS-9373 updating deploy topic.

Commit:
DOCS-9207 incorporating review comments from Sundip.

Commit:
DOCS-9207 adding link to rtfctl doc.

Commit:
DOCS-9207 adding clarification.

Commit:
DOCS-9207 adding see also link.

Commit:
DOCS-9204 add persona table.

Commit:
DOCS-9373 adding clarifiation on ingress settings.

Commit:
DOCS-9373 adding clarification on ingress tab.

Commit:
DOCS-9375 adding content for custom ingress config.

Commit:
DOCS-9375 adding custom ingress config file.

Commit:
Revising topic and product name to reflect VM bare metal.

Commit:
DOCS-9203 renaming topic to reflect current prod name.

Commit:
DOCS-9203 revising xrefs to old architecture topic.

Commit:
DOCS-9203 update topic title.

Commit:
DOCS-9203 adding information about BYOK requirements.

Commit:
DOCS-9203 adding placeholder image file.

Commit:
DOCS-9203 adding heading for architecture section.

Commit:
Adding changes from PR #307 to 1.8.

Commit:
Merge pull request #315 from mulesoft/Update-remove-node-instructions

Adding changes from PR #307 to 1.8.

Commit:
Merge pull request #310 from mulesoft/DOCS-9207-add-host-reqs

DOCS-9207 adding hosts, ports, and certs info.

Commit:
Merge pull request #311 from mulesoft/DOCS-9373-deployment

DOCS-9373 updating deploy topic.

Commit:
DOCS-9207 incorporate review comments from Sundip.

Commit:
Merge pull request #316 from mulesoft/DOCS-9207-add-host-reqs

DOCS-9207 incorporate review comments from Sundip.

Commit:
Merge pull request #312 from mulesoft/DOCS-9204-add-person-table-to-intro

DOCS-9204 add persona table.

Commit:
DOCS-9204 incorporating review comment from Sundip.

Commit:
DOCS-9375 incorporate review comments.

Commit:
DOCS-9375 updating template file. Minor edits.

Commit:
DOCS-9375 reformatting as a table.

Commit:
DOCS-9375 revising based on meeting with dev team.

Commit:
DOCS-9375 revising based on meeting with dev team.

Commit:
Merge pull request #313 from mulesoft/DOCS-9375-custom-ingress-config

DOCS-9375 adding content for custom ingress config.

Commit:
DOCS-9203 incorporate review comments from Sundip.

Commit:
Merge pull request #314 from mulesoft/DOCS-9203-architecture-limititations

DOCS-9203 updating architecture and limitations content

Commit:
DOCS-9203 incorporating review comments.

Commit:
Merge pull request #317 from mulesoft/DOCS-9203-architecture-limititations

DOCS-9203 incorporating review comments.

Commit:
DOCS-9426 removing unused topic from nav.

Commit:
DOCS-9426 moving topic to appliance only section.

Commit:
DOCS-9426 renaming topic in nav.

Commit:
DOCS-9426 remove unused file. Rename files in nav.adoc.

Commit:
DOCS-9426 removing unused file from nav.adoc.

Commit:
DOCS-9426 minor edits and terminology clarification.

Commit:
DOCS-9426 adding brief information abotu byok architecture.

Commit:
DOCS-9203 adding link from install to arch/resource topic.

Commit:
DOCS-9203 fixing link.

Commit:
DOCS-9426 adding see also links to byok topics.

Commit:
DOCS-9426 clean up and adding see also link to deployment.

Commit:
DOCS-9426 fixing link.

Commit:
DOCS-9426 fixing link.

Commit:
DOCS-9426 fixing link.

Commit:
DOCS-9426 adding version clarification. Fixing formatting.

Commit:
DOCS-9426 fixing table format.

Commit:
DOCS-9426 fixing table format.

Commit:
DOCS-9426 reformatting to avoid confusion.

Commit:
DOCS-9426 update nav.adoc.

Commit:
DOCS-9426 update nav.adoc.

Commit:
DOCS-9426 terminology clarification.

Commit:
DOCS-9426 revising nav.

Commit:
DOCS-9426 adding link to runtime RNs.

Commit:
Merge pull request #318 from mulesoft/DOCS-9426-minor-edits

DOCS-9426 minor edits and terminology clarification.

kmendiratta-crm commented on modules/ROOT/pages/install-self-managed.adoc:
```diff
@@ -0,0 +1,248 @@
+The tasks in the section must be performed by an IT administrator.
+====
+
+Runtime Fabric on Self-managed Kubernetes uses the `rtfctl` command-line utility for installation and management tasks. See xref:install-rtfctl.adoc[Install the Runtime Fabric Command Line Tool].
```
@sleslie23 there are a few new commands we have added to rtfctl that need to added to this href

kmendiratta-crm replied:
```
/rtfctl --help
rtfctl controls a Runtime Fabric cluster
	
Find more information at: https://anypoint.mulesoft.com/runtimefabric

Usage:
  rtfctl [command]

Available Commands:
  apply           Apply changes to an RTF configuration
  backup          Backup local Runtime Fabric state
  disk            Print the disk space used by an application
  get             Display one or many resources
  heapdump        Trigger a JVM heap dump
  help            Help about any command
  memory          Display the JVM memory usage in a pod
  package         Create a debugging information package for an application
  ping            Test network connectivity
  report          Generate diagnostics report
  restart         Restart an application
  restore         Restore local Runtime Fabric state from a backup
  status          Display status of the Runtime Fabric
  test            Test Runtime Fabric environment
  threaddump      Print the JVM thread dump for an application
  uninstall       Uninstall Runtime Fabric
  update          Update rtfctl
  upgrade-cluster Upgrade Runtime Fabric cluster
  validate        Validate cluster for Runtime Fabric installation
  version         Print the version information

```

sleslie23 replied:
Thanks a lot @kmendiratta-crm !!!

sleslie23 replied:
Fixed.

sundippatel committed:
Update nav.adoc

Removed RTF
Consistency w/ capitalization and terms for Self-Managed vs VMs / Bare Metal

Commit:
DOCS-9196 incorporating review comments.

Commit:
DOCS-9196 incorporate review comments.

Commit:
DOCS-9196 update xrefs.

Commit:
Fixing merge conflicts.

Commit:
Adding new rtfctl commands.
